Sources/SubscriptionTestingUtilities/APIs/APIServiceMock.swift

```diff
@@ -19,24 +19,40 @@
 import Foundation
 import Subscription
 
-public struct APIServiceMock: APIService {
-    public var mockAuthHeaders: [String: String]
+public class APIServiceMock: APIService {
```

Comment:
Minor and similar to macOS PR feedback - Might as well make these `final` by default unless not possible.